rpc/backend/tracing.go

```diff
@@ -39,9 +62,29 @@ func (b *Backend) TraceTransaction(hash common.Hash, config *evmtypes.TraceConfi
 	}
 
 	// TODO: see if we can make it work with cross chain txs
-	// check if tx is omni cross chain tx, if it is skip
+	// check if tx is omni cross chain tx, if it is return dummy data
```

Comment:
I'm okay with this for short term. But I'd like to find away to get full trace soon. this dummy data could be misleading to devs debugging transactions in explorer (when they look at internal transactions)

Reply:
Yea for future sure, it'll probably be a bigger task as tracing is a bit convoluted for ethermint. This is majorly to fix indexer log spams

Reply:
maybe for now we can update explorer ui to say that internal transactions are not supported for deposited tx. rather than showing this dummy trace